syntax = "proto3";

package message;

service VarifyService {
    // 发送email获得验证码
    rpc GetVarifyCode (GetVarifyReq) returns (GetVarifyRsp) {}
}

message GetVarifyReq {
    string email = 1;
}

message GetVarifyRsp {
    int32 error = 1;
    string email = 2;
    string code = 3;
}

service StatusService {
    // 发送uid获取chatServer的host port和token
    rpc GetChatServer (GetChatServerReq) returns (GetChatServerRsp) {}
}

message GetChatServerReq {
    int32 uid = 1;
}

message GetChatServerRsp {
    int32 error = 1;
    string host = 2;
    string port = 3;
    string token = 4;
}